osu.Game/Screens/Select/BeatmapInfoWedgeV2.cs

```diff
@@ -0,0 +1,346 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using osuTK;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Effects;
+using osu.Game.Beatmaps;
+using osu.Game.Beatmaps.Drawables;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Localisation;
+using osu.Game.Configuration;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Rulesets;
+using osu.Game.Rulesets.Mods;
+
+namespace osu.Game.Screens.Select
+{
+    public partial class BeatmapInfoWedgeV2 : VisibilityContainer
+    {
+        private const float shear_width = 21;
+        private const float wedge_height = 120;
+        private const float transition_duration = 250;
+        private const float corner_radius = 10;
+        private const float colour_bar_width = 30;
+
+        /// Todo: move this const out to song select when more new design elements are implemented for the beatmap details area, since it applies to text alignment of various elements
+        private const float text_margin = 62;
+
+        private static readonly Vector2 wedged_container_shear = new Vector2(shear_width / wedge_height, 0);
+
+        [Resolved]
+        private IBindable<RulesetInfo> ruleset { get; set; } = null!;
+
+        [Resolved]
+        private OsuColour colours { get; set; } = null!;
+
+        protected Container? DisplayedContent { get; private set; }
+
+        protected WedgeInfoText? Info { get; private set; }
+
+        private readonly Container difficultyColourBar;
+        private readonly StarCounter starCounter;
+        private readonly BufferedContainer bufferedContent;
+
+        public BeatmapInfoWedgeV2()
+        {
+            Height = wedge_height;
+            Shear = wedged_container_shear;
+            Masking = true;
+            EdgeEffect = new EdgeEffectParameters
+            {
+                Colour = Colour4.Black.Opacity(.25f),
+                Type = EdgeEffectType.Shadow,
+                Radius = corner_radius,
+                Roundness = corner_radius
+            };
+            CornerRadius = corner_radius;
+
+            // We want to buffer the wedge to avoid weird transparency overlaps between the colour bar and the background.
+            Child = bufferedContent = new BufferedContainer
```

Comment:
No, you really do not want to buffer the whole thing. Have you run the tests and looked at the quality of things in general? Using a buffered container on things like text causes texel misalignment and makes everything more blurry. There's a reason on the original info wedge it was only applied to the background.

You can work around this by using 

```suggestion
            Child = bufferedContent = new BufferedContainer(pixelSnapping: true)
```

but note that if the wedge is ever moved around on the screen using a transform, this will cause it to look less smooth.

Keep in mind there's an inherent overhead with every usage of `BufferedContainer`, too.

Reply:
@mk56-spn did you miss this feedback?

Reply:
> @mk56-spn did you miss this feedback?

Nope. Working on a fix. Been a tad busy. All should he addressed and done within the day - 2days max

Reply:
fixed, i do see a path forward which does not require pixel snapping by separating the text and background into container only one of which is nested in the buffered container, i however did not opt for this since i felt that the pixel snapping look good here ( i tried it on the carousel to get a feel for the different applications and it look god awful because it can be moved by the user and it looked choppy as hell ) and benefited the wedge anyhow by making the line between the ``colourbar`` and the background less muddy